singularity-eos/eos/shifted_eos.hpp

```diff
@@ -0,0 +1,156 @@
+//------------------------------------------------------------------------------
```

Comment:
What about a separate subdir for modifiers? Since it's not necessarily obvious that e.g. `eos_unitsystem.hpp` and `shifted_eos.hpp` are both modifiers

Reply:
Yeah this makes sense to me.